app/views/content/returning-to-teaching.md

```diff
@@ -130,25 +138,26 @@ $q-helen-returner$
 
 [Read Helen's top tips for returning teachers](/my-story-into-teaching/returners/top-tips-for-returning-teachers).
 
-## National Tutoring Programme
+## If you've been out of teaching for a while
 
-The National Tutoring Programme supports schools to close the attainment gap in light of the coronavirus pandemic and provides different options to return to a school environment without taking on a full-time or part-time classroom role.
+If you’re returning to teaching, you might want to re-familiarise yourself with life in a classroom. Here are lots of ways that you can do it.
 
-If you’re unsure about taking on a teacher role at this time, [consider applying to tutor or mentor through the National Tutoring Programme](https://nationaltutoring.org.uk/) and help provide support to disadvantaged pupils.
+#### Reading teachers' stories and tips
 
-## If you've been out of teaching for a while
+* browse [inspirational stories](https://getintoteaching.education.gov.uk/blog/tag/returning-to-teaching) from Stephen, Farooq and Maria who returned to the classroom successfully after a break
+* read up on [teaching interview tips](https://www.teachertoolkit.co.uk/2019/04/04/job-interviews) and get [pointers on remote interviews](https://www.tes.com/news/coronavirus-10-tips-acing-remote-job-interview)
```

Comment:
TES link is going to a 404?

Reply:
will ask Sharon and hopefully get it changed

Reply:
Changed this now :)